db/seeds.rb

```diff
@@ -7,3 +7,276 @@
+  finder_attributes = attributes.slice(*required_columns)
+
+  if finder_attributes.blank?
+    raise "Seeder must provide at least one of (#{finder_attributes.join(', ')}) when making an archived intake"
+  end
+
+  required_columns.each do |column|
+    if finder_attributes[column].blank?
+      raise "Seeder must provide #{column} when making an archived intake"
+    end
+  end
+
+  if attributes[:email_address].blank? && attributes[:phone_number].blank?
+    raise "Seeder must provide contact information (email/phone) when making an archived intake"
+  end
+
+  if attributes[:contact_preference] == "email" && attributes[:email_address].blank?
+    raise "Seeder must provide email_address when making an archived intake with email contact preference"
+  end
+
+  if attributes[:contact_preference] == "text" && attributes[:phone_number].blank?
+    raise "Seeder must provide phone_number when making an archived intake with text contact preference"
+  end
+end
+
+def find_or_create_az_archived_intake(attributes)
+  find_or_create_state_file_archived_intake(
+    attributes.merge(
+      state_code: "az",
+      mailing_state: "AZ",
+      fake_address_1: "685 West Cove St, Florence, AZ,  85132",
+      fake_address_2: "52 Blue Hills Drive, Phoenix, AZ, 85003"
+    )
+  )
+end
+
+def find_or_create_ny_archived_intake(attributes)
+  find_or_create_state_file_archived_intake(
+    attributes.merge(
+      state_code: "ny",
+      mailing_state: "NY",
+      fake_address_1: "1161 Rhinelander Ave, Bronx, NY, 10461",
+      fake_address_2: "921 Washington Ave, Brooklyn, NY, 11225"
+    )
+  )
+end
+
+def find_or_create_md_archived_intake(attributes)
+  find_or_create_state_file_archived_intake(
+    attributes.merge(
+      state_code: "md",
+      mailing_state: "MD",
+      fake_address_1: "212 S Market St, Frederick, MD, 21701-6527",
+      fake_address_2: "2105 Harwood Rd, District Hts, MD, 20747-2428"
+    )
+  )
+end
+
+def find_or_create_id_archived_intake(attributes)
+  find_or_create_state_file_archived_intake(
+    attributes.merge(
+      state_code: "id",
+      mailing_state: "ID",
+      fake_address_1: "11522 N Steeldust Ct, Rathdrum, ID, 83858-6714",
+      fake_address_2: "12082 W Edna St, Boise, ID, 83713-3645"
+    )
+  )
+end
+
+def find_or_create_nc_archived_intake(attributes)
+  find_or_create_state_file_archived_intake(
+    attributes.merge(
+      state_code: "nc",
+      mailing_state: "NC",
+      fake_address_1: "70 Willoughby Park Rd, King, NC, 27021",
+      fake_address_2: "3 Lark Springs Dr, Sandy Ridge, NC, 27046"
+    )
+  )
+end
+
+def find_or_create_nj_archived_intake(attributes)
+  find_or_create_state_file_archived_intake(
+    attributes.merge(
+      state_code: "nj",
+      mailing_state: "NJ",
+      fake_address_1: "6 Denver Lane, Camden, NJ, 08106",
+      fake_address_2: "190 North Jefferson Street, Hackensack, NJ, 07604"
+    )
+  )
+end
+
+if ENV["REVIEW_APP"] == "true"
```

Comment:
Need to set this on `staging` and `heroku` review apps
1. heroku: set on `app.json`
2. staging: set on pya module